tapalcatl_server/storage.go

```diff
@@ -120,6 +120,13 @@ func (s *S3Storage) Fetch(t tapalcatl.TileCoord, c Condition) (*StorageResponse,
 	return result, nil
 }
 
+func (s *S3Storage) healthCheck() (error) {
+	key := "sanity_check.txt"
```

Comment:
The configuration is parsed from JSON input using the [`storageDefinition`](https://github.com/tilezen/tapalcatl/blob/bb4cde92739f6d76b9b0ca03906e1d6f90b9ebd1/tapalcatl_server/server.go#L34) and [`storageConfig`](https://github.com/tilezen/tapalcatl/blob/bb4cde92739f6d76b9b0ca03906e1d6f90b9ebd1/tapalcatl_server/server.go#L57) `struct`s. The convention seems to be that the `storageConfig` overrides the `storageDefinition`, for example when setting the [key pattern](https://github.com/tilezen/tapalcatl/blob/bb4cde92739f6d76b9b0ca03906e1d6f90b9ebd1/tapalcatl_server/server.go#L432-L435).

While I'm sure there's a Better Way to Do It, the simplest thing for now would be to add another field alongside `KeyPattern` in those two `struct`s, add [docs](https://github.com/tilezen/tapalcatl/blob/bb4cde92739f6d76b9b0ca03906e1d6f90b9ebd1/tapalcatl_server/server.go#L311) and override the config, passing the value to the storage constructors where they can use it.

Should `HealthCheck() error` also be a part of the public `Storage` interface (L35-37)?